### PR TITLE
Pull Request Guidelines

### PR DESCRIPTION
**Pull request by jackcarlisle**
### ready for review
- adds guidelines for creating a PR 
  #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,6 +74,13 @@ Example user story: `As an admin I want to be able to create new users because I
 
 _These guidelines for creating user stories are an adaptation of [Simon's](https://github.com/simonlab) [**user-story-checklist**](https://github.com/SimonLab/user-story-checklist) repo. Check it out!_
 
+### Pull Requests
+
+At dwyl we like to work in the most ***agile*** way possible wherever we can. This even extends to how we submit and review our **pull requests**. You might have the tendency, if you have been working on a project, to bundle up all of your bug fixes/feature additions into one large PR and then submit it to be reviewed all at once. At dwyl we prefer working in smaller increments splitting the work into individual parts to be reviewed separately. We prefer this for a couple of reasons:
+
++ It allows us to keep track of the work more easily because it provides more descriptive reference points (paper trail) to which we can look back at in future if required
++ It's gives the person who is reviewing the PR to focus on one thing at a time which improves the quality of the review
```
Comment (nelsonic) [inline README.md]: @jackcarlisle this sentence does not read well. maybe try:

> `It gives the person reviewing the PR a single focus for the change/bug-fix which improves review speed.`

File: README.md [hunk 2]
```diff
@@ -74,6 +74,13 @@ Example user story: `As an admin I want to be able to create new users because I
 
 _These guidelines for creating user stories are an adaptation of [Simon's](https://github.com/simonlab) [**user-story-checklist**](https://github.com/SimonLab/user-story-checklist) repo. Check it out!_
 
+### Pull Requests
```
Comment (iteles) [inline README.md:77]: Oooooooh, this whole section actually reminds me of: https://github.com/dwyl/style-guide#pull-requests

I'm not even sure which repo all of this should be in... @nelsonic ?

Comment (jackcarlisle) [reply README.md:77]: I can include this PR info in the contributing repo if you like?

Comment (nelsonic) [reply README.md:77]: @iteles I _think_ the [`style-guide`](https://github.com/dwyl/style-guide#pull-requests) should _briefly_ address the **commit messages** and _link_ to `contributing` (_this repo_) which should include _all_ the detail. Preferably with pictures/examples.

**Comment by nelsonic** [comment]
@iteles please merge if/when you're happy. 👍

**Comment by iteles** [comment]
Argh sorry, will look at this a little later this afternoon 👍

**Comment by iteles** [comment]
This is ace, thanks @jackcarlisle ! We'll take a look at consolidating between the 2 repos later 👍

**Comment by jackcarlisle** [comment]
@iteles sounds good!

**Pull request merged**